Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -499,6 +499,17 @@ extension TelemetryWrapper {
             GleanMetrics.Widget.mQuickActionClosePrivate.add()
         case (.action, .open, .mediumTopSitesWidget, _, _):
             GleanMetrics.Widget.mTopSitesWidget.add()
+        // Experiments
+        case (.enrollment, .add, .experimentEnrollment, _, let extras):
+            if let id = extras?["Experiment id"] as? String, let name = extras?["Experiment name"] as? String, let variant = extras?["Experiment variant"] as? String {
+                GleanMetrics.Experiments.experimentEnrollment.record(
+                extra: [GleanMetrics.Experiments.ExperimentEnrollmentKeys.experimentId: id,
```

Comment:
nit: wondering if there is a cleaner way of using these enums, this whole nested enum taks so much space 

GleanMetrics.Experiments.ExperimentEnrollmentKeys.experimentId